src/utils/JobRestAPIUtils.py

```diff
@@ -741,25 +723,45 @@ def GetJobStatus(jobId):
     return result
 
 
-def GetJobLog(userName, jobId):
+def GetJobLog(userName, jobId, cursor=None, size=100):
     dataHandler = DataHandler()
     jobs = dataHandler.GetJob(jobId=jobId)
     if len(jobs) == 1:
         if jobs[0]["userName"] == userName or AuthorizationManager.HasAccess(userName, ResourceType.VC, jobs[0]["vcName"], Permission.Collaborator):
-            try:
-                log = dataHandler.GetJobTextField(jobId, "jobLog")
+            if elasticsearch_deployed:
+                (logs, cursor) = UtilsGetJobLog(jobId, cursor, size)
+
+                pod_logs = {}
+                for log in logs:
+                    try:
+                        pod_name = log["_source"]["kubernetes"]["pod_name"]
+                        log = log["_source"]["log"]
+                        if pod_name in pod_logs:
+                            pod_logs[pod_name] += log
+                        else:
+                            pod_logs[pod_name] = log
+                    except Exception:
+                        logging.exception("Failed to parse elasticsearch log: {}".format(log))
+
+                return {
+                    "log": pod_logs,
```

Comment:
This changed structure of return, if elastic search was deployed, returns a map; if not, returns a string.

Reply:
The return object is inconsistent before this PR. The default return before this PR was also:
```
return {
        "log": {},
        "cursor": None,
}
```
@Gerhut could you help fix it?

Reply:
Fixed in #824 

Not fully tested since I have no blobfuse test features.